cernopendata/modules/fixtures/data/records/cms-simulated-datasets-Run2011A.json

```diff
@@ -43843,5 +43843,107 @@
     }
+  },
+  {
+    "abstract": {
+      "description": "<p>Simulated dataset GluGluToHToGG_M-125_7TeV-powheg-pythia6 in AODSIM format for 2011 collision data.</p><p>See the description of the simulated dataset names in: <a href=\"/about/CMS-Simulated-Dataset-Names\">About CMS simulated dataset names</a>.</p>"
+    },
+    "accelerator": "CERN-LHC",
+    "categories": {
+      "primary": "Higgs Physics",
+      "secondary": [
+        "Standard Model"
+      ],
+      "source": "CMS Collaboration"
+    },
+    "collaboration": {
+      "name": "CMS Collaboration",
+      "recid": "451"
+    },
+    "collections": [
+      "CMS-Simulated-Datasets"
+    ],
+    "collision_information": {
+      "energy": "7TeV",
+      "type": "pp"
+    },
+    "date_created": "2011",
+    "date_published": "2018",
+    "date_reprocessed": "2011",
+    "distribution": {
+      "formats": [
+        "aodsim",
+        "root"
+      ],
+      "number_events": 99988,
+      "number_files": 7,
+      "size": 21339915896
+    },
+    "experiment": "CMS",
+    "files": [
+      {
+        "checksum": "adler32:da5e71c4",
+        "description": "GluGluToHToGG_M-125_7TeV-powheg-pythia6 AOD dataset file index (1 of 1) for access to data via CMS virtual machine",
+        "size": 2291,
+        "type": "index.json",
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/MonteCarlo2011/Fall11/GluGluToHToGG_M-125_7TeV-powheg-pythia6/AODSIM/PU_S6_START44_V9B-v1/file-indexes/CMS_MonteCarlo2011_Fall11_GluGluToHToGG_M-125_7TeV-powheg-pythia6_AODSIM_PU_S6_START44_V9B-v1_0000_file_index.json"
+      },
+      {
+        "checksum": "adler32:6f328076",
+        "description": "GluGluToHToGG_M-125_7TeV-powheg-pythia6 AOD dataset file index (1 of 1) for access to data via CMS virtual machine",
+        "size": 1260,
+        "type": "index.txt",
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/MonteCarlo2011/Fall11/GluGluToHToGG_M-125_7TeV-powheg-pythia6/AODSIM/PU_S6_START44_V9B-v1/file-indexes/CMS_MonteCarlo2011_Fall11_GluGluToHToGG_M-125_7TeV-powheg-pythia6_AODSIM_PU_S6_START44_V9B-v1_0000_file_index.txt"
+      }
+    ],
+    "license": {
+      "attribution": "CC0"
+    },
+    "note": {
+      "description": "These simulated datasets correspond to the collision data collected by the CMS experiment in 2011."
+    },
+    "pileup": {
+      "description": " <p> To make these simulated data comparable with the collision data, <a href=\"/docs/cms-guide-pileup-simulation\">pile-up events</a> are added to the simulated event in this step. </p> ",
+      "links": [
+        {
+          "recid": "3600",
+          "title": "/MinBias_TuneZ2_7TeV-pythia6/Summer11Leg-START53_LV4-v1/GEN-SIM"
+        }
+      ]
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "1681",
+    "run_period": [
+      "Run2011A",
+      "Run2011B"
+    ],
+    "system_details": {
+      "global_tag": "START53_LV6A1::All",
```

Comment:
Shall we use `_LV6` here as the rest of 2011 datasets?

Reply:
The recommended one is `START53_LV6A1::All`. See http://opendata.cern.ch/docs/cms-guide-for-condition-database

Reply:
Let us update it elsewhere too then...